translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\n"
+"\t\tA chave pública/privada deve existir antes. O certificado da chave "
+"pública deve ser .PEM encoded e ter sido generado pelo a chave privada."
+
+#: pkg/kubectl/cmd/create_configmap.go:32
+msgid ""
+"\n"
+"\t\tCreate a configmap based on a file, directory, or specified literal "
+"value.\n"
+"\n"
+"\t\tA single configmap may package one or more key/value pairs.\n"
+"\n"
+"\t\tWhen creating a configmap based on a file, the key will default to the "
+"basename of the file, and the value will\n"
+"\t\tdefault to the file content.  If the basename is an invalid key, you may "
+"specify an alternate key.\n"
+"\n"
+"\t\tWhen creating a configmap based on a directory, each file whose basename "
+"is a valid key in the directory will be\n"
+"\t\tpackaged into the configmap.  Any directory entries except regular files "
+"are ignored (e.g. subdirectories,\n"
+"\t\tsymlinks, devices, pipes, etc)."
+msgstr ""
+"\n"
+"\t\tCria um configmap com base em um arquivo, diretório, ou um valor literal "
+"especificado.\n"
+"\n"
+"\t\tUm confimap único pode conter um ou mais pares de chave/valor.\n"
+"\n"
+"\t\tQuando criar um configmap com base em um arquivo, a chave será por "
+"padrão o nome do arquivo, e o valor será\n"
+"\t\tpor padrão o conteúdo do arquivo.  Se o nome do arquivo for uma chave "
+"inválida, você deve especificar uma chave alternativa.\n"
+"\n"
+"\t\tQuando criar um configmap com base em um diretório, cada arquivo cujo o "
+"nome é uma chave válida no diretório será\n"
+"\t\tcolocada no configmap.  Qualquer entrada de diretório, exceto as com "
+"arquivos válidos serão ignorados (por exemplo: sub-diretórios,\n"
+"\t\tsymlinks, devices, pipes, etc)."
+
+#: pkg/kubectl/cmd/create_namespace.go:32
+msgid ""
+"\n"
+"\t\tCreate a namespace with the specified name."
+msgstr ""
+"\n"
+"\t\tCria um namespace com um nome especificado."
+
+#: pkg/kubectl/cmd/create_secret.go:119
+msgid ""
+"\n"
+"\t\tCreate a new secret for use with Docker registries.\n"
+"\n"
+"\t\tDockercfg secrets are used to authenticate against Docker registries.\n"
+"\n"
+"\t\tWhen using the Docker command line to push images, you can authenticate "
+"to a given registry by running\n"
+"\n"
+"\t\t    $ docker login DOCKER_REGISTRY_SERVER --username=DOCKER_USER --"
+"password=DOCKER_PASSWORD --email=DOCKER_EMAIL'.\n"
+"\n"
+"    That produces a ~/.dockercfg file that is used by subsequent 'docker "
+"push' and 'docker pull' commands to\n"
+"\t\tauthenticate to the registry. The email address is optional.\n"
+"\n"
+"\t\tWhen creating applications, you may have a Docker registry that requires "
+"authentication.  In order for the\n"
+"\t\tnodes to pull images on your behalf, they have to have the credentials.  "
+"You can provide this information\n"
+"\t\tby creating a dockercfg secret and attaching it to your service account."
+msgstr ""
+"\n"
+"\t\tCria um novo secret para utilizar com o Docker registries.\n"
```

Comment:
you can remove the "o" before Docker registries, as docker registries here is plural